docs/sidebars.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  someSidebar: {
+    General: ["quick_start", "safe_migrations", "cli", "github_app"],
+    Rules: [
+      "adding-field-with-default",
```

Comment:
what if we put all the rules related docs in a `//docs/rules/` directory and generated this array by `ls`ing the dir?

Reply:
seems like more work. Maybe something to add later?